objax/module.py

```diff
@@ -143,6 +147,17 @@ def __getitem__(self, key: Union[int, slice]):
             return ModuleList(value)
         return value
 
+    def __repr__(self):
+        def f(x):
+            if not isinstance(x, Module) and callable(x):
+                return repr_function(x)
+            x = repr(x).split('\n')
+            x = [x[0]] + ['  ' + y for y in x[1:]]
```

Comment:
Will this work correctly for multiple levels of nesting of Sequential?
It looks like it only adds two spaces in front of lines here and then concatenates everything using newline.

Could you add test of with several level of depth of Sequential to make sure that it works, i.e.:
```
m = objax.Sequential(
   op11,
   objax.Sequential(
     op21,
     objax.Sequential(
       op31,
       op32,
       objax.Sequential(op41, op42),
     )
   )
   op12
)
```

Reply:
Yes it does, there's a unit test for it.